added-seo-to-qts-page (#4279)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/what-is-qts.md

You can get QTS through undergraduate or postgraduate initial teacher training. This mostly involves school placements with some theoretical learning.

> You can apply for either primary teaching with QTS or a secondary teacher training course awarding QTS. [Find out more about the age groups you could teach](/life-as-a-teacher/age-groups-and-specialisms/age-groups-you-could-teach).

Reviewer: ```suggestion
You can apply for courses that offer either primary teaching with QTS or secondary teaching with QTS. [Find out more about the age groups you could teach](/life-as-a-teacher/age-groups-and-specialisms/age-groups-you-could-teach).
```

Reviewer: Just amended this sightly as it sounded a bit like they were two totally different things

If you trained to teach in Wales, your QTS will be automatically recognised in England and awarded by the [Education Workforce Council (EWC)](https://www.ewc.wales/site/index.php/en/).

> If you trained to teach in Scotland or Northern Ireland, you can [apply for QTS in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start). If your QTS application is successful, you will gain QTS without any further fees or training.

Reviewer: ```suggestion
If you trained to teach in Scotland or Northern Ireland, you can [apply for QTS in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start). If your application is successful, you will gain QTS without any further fees or training.
```
